app/src/routes/v3/forest-watcher.router.js

```diff
@@ -149,6 +149,32 @@ class ForestWatcherRouter {
     };
   }
 
+  static async getUserTeamsAreas(ctx) {
+    const user = ForestWatcherFunctions.getUser(ctx);
+    let data = [];
+    if (user && user.id) {
+      try {
+        const userAreas = await AreasService.getUserAreas(user.id);
+        // get a users teams
+        const userTeams = await TeamService.getUserTeams(user.id); // get list of user's teams
```

Comment:
Is getUserTeams() using `/v3/teams/user/{userId}` ?

Reply:
Yes